bindings/nodejs/lib/utils/utils.ts

```diff
@@ -53,6 +57,24 @@ export class Utils {
         });
     }
 
+    /**
+     * Computes the foundry id.
+     */
+    static computeFoundryId(
+        aliasId: AliasId,
+        serialNumber: number,
+        tokenSchemeKind: number,
+    ): string {
```

Comment:
Shouldn't this be a FoundryId return type?

Reply:
add3ed alias and updated